src/Time/Units.hs

```diff
@@ -100,6 +114,41 @@ instance KnownSymbol (ShowUnit unit) => Show (Time unit) where
                                                 n -> '/' : show n
                       in numeratorStr ++ denominatorStr ++ symbolVal (Proxy @(ShowUnit unit))
 
+----------------------------------------------------------------------------
+-- Creation helpers
+----------------------------------------------------------------------------
+
+sec :: Natural -> Second
+sec n = Time (n % 1)
+
+ms :: Natural -> MilliSecond
+ms n = Time (n % 1)
```

Comment:
It's better to use `:%` constructor directly here, because `%` function calculates `gcd` and normalizes rational numbers. We don't need normalization, thus, functions can be faster.